Duplicati/Server/webroot/ngax/scripts/services/EditUriBuiltins.js

```diff
@@ -487,22 +487,42 @@ backupApp.service('EditUriBuiltins', function (AppService, AppUtils, SystemInfo,
     }
 	
 	EditUriBackendConfig.parsers['tardigrade'] = function (scope, module, server, port, path, options) {
-        if (options['--tardigrade-auth-method'])
-            scope.tardigrade_auth_method = options['--tardigrade-auth-method'];
+		if (options['--tardigrade-auth-method'])
+            scope.storj_auth_method = options['--tardigrade-auth-method'];
         if (options['--tardigrade-satellite'])
-            scope.tardigrade_satellite = options['--tardigrade-satellite'];
+            scope.storj_satellite = options['--tardigrade-satellite'];
         if (options['--tardigrade-api-key'])
-            scope.tardigrade_api_key = options['--tardigrade-api-key'];
+            scope.storj_api_key = options['--tardigrade-api-key'];
         if (options['--tardigrade-secret'])
-            scope.tardigrade_secret = options['--tardigrade-secret'];
+            scope.storj_secret = options['--tardigrade-secret'];
+		if (options['--tardigrade-secret-verify'])
+            scope.storj_secret_verify = options['--tardigrade-secret-verify'];
 		if (options['--tardigrade-shared-access'])
-            scope.tardigrade_shared_access = options['--tardigrade-shared-access'];
+            scope.storj_shared_access = options['--tardigrade-shared-access'];
 		if (options['--tardigrade-bucket'])
-            scope.tardigrade_bucket = options['--tardigrade-bucket'];
+            scope.storj_bucket = options['--tardigrade-bucket'];
 		if (options['--tardigrade-folder'])
-            scope.tardigrade_folder = options['--tardigrade-folder'];
-
-        var nukeopts = ['--tardigrade-auth-method','--tardigrade-satellite', '--tardigrade-api-key', '--tardigrade-secret', '--tardigrade-shared-access', '--tardigrade-bucket', '--tardigrade-folder'];
+            scope.storj_folder = options['--tardigrade-folder'];
+		
+        if (options['--storj-auth-method'])
+            scope.storj_auth_method = options['--storj-auth-method'];
+        if (options['--storj-satellite'])
+            scope.storj_satellite = options['--storj-satellite'];
+        if (options['--storj-api-key'])
+            scope.storj_api_key = options['--storj-api-key'];
+        if (options['--storj-secret'])
+            scope.storj_secret = options['--storj-secret'];
+		if (options['--storj-secret-verify'])
```

Comment:
Can we fix the indentation here (spaces vs. tabs)?

Reply:
Fixed